app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/messages/MessagesViewModel.java

```diff
@@ -0,0 +1,95 @@
+package com.teamagam.gimelgimel.app.model.ViewsModels.messages;
+
+import com.teamagam.gimelgimel.app.common.DataChangedObservable;
+import com.teamagam.gimelgimel.app.common.DataChangedObserver;
+import com.teamagam.gimelgimel.app.common.NotifyingDataChangedObservable;
+import com.teamagam.gimelgimel.app.model.ViewsModels.Message;
+import com.teamagam.gimelgimel.app.model.entities.messages.MessagesModel;
+import com.teamagam.gimelgimel.app.model.entities.messages.MessagesReadStatusModel;
+import com.teamagam.gimelgimel.app.model.entities.messages.SelectedMessageModel;
+
+/**
+ * Messages view-model for messages presentation use-case
+ */
+public class MessagesViewModel extends NotifyingDataChangedObservable implements DataChangedObservable {
```

Comment:
But its not a base viewmodel class.
" I dont like" is not a valid argument. !resolved